src/format/parse.rs

```diff
@@ -424,27 +451,24 @@ where
                     | &TimezoneOffsetDoubleColon
                     | &TimezoneOffsetTripleColon
                     | &TimezoneOffset => {
-                        let offset = try_consume!(scan::timezone_offset(
-                            s.trim_left(),
-                            scan::colon_or_space
-                        ));
+                        s = scan::space1(s);
+                        let offset = try_consume!(scan::timezone_offset(s, scan::colon_or_space));
                         parsed.set_offset(i64::from(offset)).map_err(|e| (s, e))?;
                     }
 
                     &TimezoneOffsetColonZ | &TimezoneOffsetZ => {
-                        let offset = try_consume!(scan::timezone_offset_zulu(
-                            s.trim_left(),
-                            scan::colon_or_space
-                        ));
+                        s = scan::space1(s);
+                        let offset =
+                            try_consume!(scan::timezone_offset_zulu(s, scan::colon_or_space));
```

Comment:
should we (here or in another PR) consider being explicit about all the parts of the offset (eg offset hours, offset minutes)? As far as I can see in the strftime docs we don't mention being permissive with space and/or colons in `%z`, `%:z` or `%#z`

Reply:
Yes, definitely. It would help users avoid unwanted surprises.
How about I first create a tracking Issue around this?